src/main/java/org/launchcode/liftoffproject/models/Child.java

```diff
@@ -64,22 +78,6 @@ public void setLastName(String lastName) {
         this.lastName = lastName;
     }
 
-    public int getPoints() {
-        return points;
-    }
```

Comment:
we need the get/setParent aspects to make sure we can link child/parents together. I think we should just use the existing "getPoints" and "setPoints" instead of the "earnedRewardsPoints" since they should be representing the same thing